user-page/user_homepage/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+    <meta charset="utf-8">
+    <title>Attendace system</title>
+    <link rel="stylesheet" href="master.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
+    <script src="jquery-3.4.1.min.js"></script>
+</head>
+
+<body>
+    <header>
+        <nav>
+            <h1>Welcome to attendace system!</h1>
+            <button id="menuButton">menu</button>
```

Comment:
Do we really want to have a menu for the user? We didn't plan this.

Reply:
Yes we do. I am not planning on full development of dropdown menu. Its just a placeholder for now.